satellite-clone-vars.sample.yml

```diff
@@ -65,7 +65,9 @@
 #restorecon: false
 
 # Disable checking that satellite is already installed. (defaults to false)
-#skip_satellite_rpm_check: false
+# Once the playbook confirms that there is no satellite installation, the subsequent runs will not run this task.
+# To force run this task for a specific run, use `--flush-cache` option
```

Comment:
I'm fine with changing the variable name for consistency, but I think this explanation is overly confusing from a user perspective. The intent behind having the check only run on the first run is so the user doesn't have to worry about it. If they do hit the satellite rpm check unnecessarily, the error message is pretty clear about what to do to work around the blockage. I view this as some complexity a user shouldn't have to deal with unless they have an issue with it.

That being said, if you want to add comments to the code for our own sanity, I'm all for that :smile: